back-end/app/Http/Controllers/User/UsersController.php

```diff
@@ -128,17 +129,41 @@ public function login(Request $request)
             return response()->json(['message' => 'Email or Password is incorrect'], 401);
         }
 
+        $user->update([
+            "is_active" => true,
+        ]);
+
+        $images = Image::where('user_id', $user->id)->first();
+
+        $userData = [
+            'id' => $user->id,
+            'first_name' => $user->first_name,
+            'last_name' => $user->last_name,
+            'email' => $user->email,
+            'is_active' => $user->is_active,
+            'is_admin' => $user->is_admin,
+            'avatar_url' => $images->avatar_url,
+            'cover_url' => $images->cover_url
+        ];
+
         $token = $user->createToken('elstoken')->plainTextToken;
 
         return response()->json([
             'message' => 'Success Login',
-            $token
+            'token' => $token,
+            'data' => $userData,
         ], 201);
     }
 
-    // Logout user and delete the access token from storage. 
-    public function logout()
+    // Logout user and delete the access token from storage and set is_active to false. 
+    public function logout(Request $user_id)
     {
+        $user = User::findOrFail($user_id);
+
+        $user[0]->update([
+            "is_active" => false,
+        ]);
```

Comment:
My plan for is active is when it is true, the UI for the users will have a border color indicating that they're active.
But the main goal for that is in the authentication, in my hooks that's the one that the hook is checking so that the users cannot modify the cookies and get inside the dashboard when they're not online.